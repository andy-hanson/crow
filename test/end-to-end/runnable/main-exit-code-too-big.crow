main nat(_ string[])
	max-value	
