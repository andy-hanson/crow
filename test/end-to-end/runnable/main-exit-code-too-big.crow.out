exit code: big
stderr:
'main' must return a number <= 2147483647; got 18446744073709551615
